apps/HelloPyTorch/Makefile

```diff
@@ -12,30 +12,36 @@ CUDA_OPS =
 else
 HAS_CUDA = 1
 CUDA_OPS = $(BIN)/$(HL_TARGET)/add_cuda_float32.a \
-	       $(BIN)/$(HL_TARGET)/add_cuda_float64.a \
+	   $(BIN)/$(HL_TARGET)/add_cuda_float64.a \
            $(BIN)/$(HL_TARGET)/add_grad_cuda_float32.a \
-	       $(BIN)/$(HL_TARGET)/add_grad_cuda_float64.a
+	   $(BIN)/$(HL_TARGET)/add_grad_cuda_float64.a 
 endif
 
+EXT_LIB=$(BIN)/lib
 PYTHON ?= python3
+PIP ?= pip3
 
 all: test
 
 # Run the PyTorch tests to verify the module is compiled correctly
 # .wrapper is a dummy file whose timestamps allow Make to track the dependency
-# on the Python side of the build
-test: .wrapper
-	@PYTHONPATH=lib:${PYTHONPATH} $(PYTHON) test.py
+# on the Python side of the build 
+
+HALIDE_DISTRIB_PATH=$(shell pwd)/../../distrib
```

Comment:
HALIDE_DISTRIB_PATH should already be defined in Makefile.inc... if that definition isn't working, we should probably fix it there rather than redefine it here

Reply:
So, the issues comes up in the `pip install` build: the build system creates a temp directory and since the path is relative, it looks for Halide in the wrong directory. Making HALIDE_DISTRIB_PATH absolute would fix it.

Reply:
In that case I suspect that the best fix is changing Makefile.inc to use `realpath`:

    HALIDE_DISTRIB_ATH ?= $(realpath ../../distrib)

Reply:
I would agree for linux machines, but I think e.g. `realpath` is not installed by default on OSX..

Reply:
`realpath` as a bash command is not available on OSX, but `$(realpath)` is part of standard make ~everywhere, and works fine on OSX (we already use it in `Makefile`)

Reply:
fixed in 93c2db901